src/fiber.cr

```diff
@@ -67,6 +35,7 @@ class Fiber
   end
 
   def initialize(@name : String? = nil, &@proc : ->)
+    @context = Context.new
```

Comment:
I would expect that each `Fiber#initialize` would set `@context.resumable` explicitly at least on one of the overloads.

Currently, all `Fiber#makecontext` implementations are doing `@context.resumable = 1` which seems a bit odd.

Reply:
I'm not sure. I meant for the context to be manipulated by the arch-specific functions, and `makecontext` is responsible for telling when the context is actually resumable. I'm not sure Fiber should assume anything about it's resumable state (only about being alive/dead).

Reply:
Ok, let's keep it as is.